src/androidTest/java/com/nutomic/syncthingandroid/test/syncthing/RestApiTest.java

```diff
@@ -75,7 +73,7 @@ public void onReceiveSystemInfo(RestApi.SystemInfo info) {
     public void testGetFolders() {
         assertNotNull(mApi.getFolders());
     }
-    
+
     public void testConvertNotCrashing() {
```

Comment:
Edit 3:
I just confirmed: Android Studio (at least in Version 2.0) automatically made those changes without notifying me. Simply viewing the file in the editor is enough